ATTRIBUTION.md

```diff
@@ -0,0 +1,5 @@
+# static/feather-sprite.svg
+
+Downloaded from https://unpkg.com/feather-icons/dist/feather-sprite.svg,
+following this guide: https://github.com/feathericons/feather#svg-sprite.
+See license here: https://github.com/feathericons/feather/blob/master/LICENSE
```

Comment:
[These icons](https://github.com/feathericons/feather) are licensed under an MIT license. I'm not sure whether we can pull them into this repo, since it's a different license. This is my attempt at doing it while giving proper attribution.

Alternatively, we could also not use these icons, of course. Although they're meant to be freely used, so hopefully we can figure this out.

Reply:
I think that's okay. The things that we author are licensed under the CC0, and things that we don't author are available under their respective licenses. The work as a whole is subject to the union of the restrictions of the licenses of the parts, but each separately licensed component of the work can be separated out and used/redistributed/modified according to the terms of that part's license.

This is already the case for our dependencies, all of which are currently MIT, or MIT / Apache 2.0. (You can use `cargo-lichking` to see licenses, `cargo install cargo-lichking && cargo lichking list`.)

Most commonly used licenses don't have any restrictions that we'd have trouble satisfying. (E.g. some require that you include the license, or that you attribute the author, which are all pretty easy.) The only licenses that we should watch out for are the GPL and the AGPL. If we have a GPL or AGPLed dependency, things get more complicated.

I'm usually pretty lax about attribution in projects, because nobody really cares or enforces it, but I think that adding an `ATTRIBUTION.md` as you've done is the scrupulously correct thing to do.

Reply:
Yeah, I didn't even think about our rust dependencies. And in the the release binaries both rust dependencies and these icons are included. Sounds good then.